### DISCUSSION TITLE
Alerts are sending two duplicate emails instead of just one

### DISCUSSION THREAD

**Discussion by lymanaukuleles** [jertel/elastalert2#857] [Q&A] [answered]
Not sure the reason, but my alerts are now sending two duplicate emails once they are triggered. How can I change this so that it is back to sending only one email?

**Comment by nsano-rururu** [comment, marked as answer]
https://elastalert2.readthedocs.io/en/latest/recipes/faq.html#how-can-i-prevent-duplicate-alerts

**Reply by lymanaukuleles** [reply]
I added this to my rule file:
```
realert:
 days: 1
```
But am still getting duplicate alerts. So what else should I do?

**Reply by nsano-rururu** [reply]
Read the document
[realert](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#realert)

**Reply by lymanaukuleles** [reply]
I don't get it

**Reply by nsano-rururu** [reply]
https://github.com/Yelp/elastalert/issues/731

**Reply by nsano-rururu** [reply]
>I don't get it

Isn't it crazy?

**Reply by nsano-rururu** [reply]
I can't answer because I don't know the settings. Please solve it yourself.

**Reply by nsano-rururu** [reply]
**example**

**before**

```yaml
timeframe:
  minutes: 5
```

**after**

```yaml
timeframe:
  minutes: 5
realert:
  minutes: 5
```

**Reply by lymanaukuleles** [reply]
I managed to fix it. It actually had nothing to do with ElastAlert. It was something in my email settings.
